.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media/
```

Comment:
The `media/` directory is being ignored. If this directory is used for storing user-uploaded files or other important data, ensure you have a backup or alternative storage solution, as ignoring it in version control might lead to data loss.